src/components/DetailContainer.js

```diff
@@ -5,8 +5,10 @@ import sharedStyle from './sharedStyle.css'
 
 const DetailContainer = ({ children, ...props }) => {
   return (
-    <div className={`container-fluid ${sharedStyle['move-left-detail']} ${sharedStyle['detail-z-index']}`} {...props}>
-      {children}
+    <div className='detail-container'>
```

Comment:
Don't need a wrapper div, just add 'detail-container' to the containing div as the first style.

Reply:
It will not work as expected, because `<div className={`container-fluid` has different non-zero margin.

Reply:
Ok